src/components/PatientRegistration/PatientRegistration.tsx

```diff
@@ -1,12 +1,15 @@
 import React, { ReactElement, useState, useEffect } from 'react'
 import './patientRegistration.css'
 import Checkbox from './Checkbox';
+import { authApi } from '../../api/authApi';
 
 
 export default function PatientRegistration(): ReactElement {
   const [firstName, setName] = useState('');
   const [lastName, setLastName] = useState('');
-  const [dateBirth, setDateBirth] = useState('');
+
+  // const [startDate, setStartDate] = useState<any>(new Date());
+  const [dateBirth, setDateBirth] = useState<any>(new Date());
```

Comment:
Default birthday - today :)))

Reply:
it's really funny) why not!)))

I meant the type for this value